Assets/Scripts/Missions/MissionTutorial.cs

```diff
@@ -488,7 +491,11 @@ private void SpawnEnemies()
         {
             group.enemyStartPositions.ForEach(position =>
             {
-                GameObject enemyInstance = Instantiate(chaserPrefab, position, Quaternion.identity);
+                GameObject enemyInstance = Instantiate(chaserPrefabs[Random.Range(0, chaserPrefabs.Count)], position, Quaternion.identity);
+                enemyInstance.transform.localScale = new Vector3(
+                    enemyInstance.transform.localScale.x * Random.Range(randomWidthRange.x, randomWidthRange.y),
+                    enemyInstance.transform.localScale.y * Random.Range(randomHeightRange.x, randomHeightRange.y),
+                    enemyInstance.transform.localScale.z * Random.Range(randomWidthRange.x, randomWidthRange.y));
```

Comment:
Since we're accessing the localScale a few times anyways:

```suggestion
                Vector3 originalScale = enemyInstance.transform.localScale;
                enemyInstance.transform.localScale = new Vector3(
                    originalScale.x * Random.Range(randomWidthRange.x, randomWidthRange.y),
                    originalScale.y * Random.Range(randomHeightRange.x, randomHeightRange.y),
                    originalScale.z * Random.Range(randomWidthRange.x, randomWidthRange.y));
```